packages/psd/src/classes/Group.ts

```diff
@@ -68,4 +70,30 @@ export class Group implements NodeBase<NodeParent, NodeChild> {
   get id(): number | undefined {
     return this.layerFrame?.id;
   }
+
+  async userMask(): Promise<Uint8Array | undefined> {
+    const userMask = this.layerFrame?.userMask;
+
+    if (!userMask) {
+      return undefined;
```

Comment:
Would be nice to have some consistency here between `return` vs `return undefined` ;)

Reply:
removed undefined